### PR TITLE
Bk day 03 code revise

### PR DESCRIPTION
**Pull request by evanyeohboonkhai**

### CODE REVIEW

**Comment by evanyeohboonkhai** [comment]
Day 3 code revise complete : 

Revise : 
- 03 - Data Preparation for Deep Learning.ipynb

Files that move to the achieved folder : 
Reason: Dataset is not found
- 03 - Multivariate Forecasting (Direct Multistep Forecast).ipynb
- 03 - Multivariate Forecasting (Multi Output Forecast).ipynb
- 03 - Multivariate Forecasting (Recursive Multistep Forecast).ipynb

File: notebook/Solution/03 - Data Preparation for Deep Learning.ipynb [hunk 1]
```diff
@@ -1104,69 +1184,45 @@
...
-      "\u001b[1;31mIndexError\u001b[0m: tuple index out of range"
+      "feature shape: (6, 3)\n",
+      "label shape: (6, 2)\n"
      ]
     }
    ],
    "source": [
     "# call function to perform sequencing\n",
-    "X_feature, Y_label = univariate_multi_step(univariate_series,window_size = 3,n_multistep = 2)\n",
+    "x_feature, y_label = univariate_multi_step(univariate_series,window_size = 3,n_multistep = 2)\n",
```
Comment (KianYang-Lee) [inline notebook/Solution/03 - Data Preparation for Deep Learning.ipynb:1208]: the usual naming convention is either "X" or "feature"

**Review by KianYang-Lee — APPROVED** [review]
1. Might want to consider the archival of the three notebooks. Dataset is there if you look closely.

**Review by scot-certifai — APPROVED** [review]
I believe this is good to merge.

**Comment by evanyeohboonkhai** [comment]
> I believe this is good to merge.

Please do not merge it first, i will add an extra notebook for Data Preprocessing

**Review by scot-certifai — APPROVED** [review]
Looks good.

**Comment by scot-certifai** [comment]
solved conflicts.

**Comment by evanyeohboonkhai** [comment]
add new 03 - Data Preprocessing Techniques.ipynb note , please have view on it , thanks @scot-certifai @KianYang-Lee

**Review by scot-certifai — APPROVED** [review]
From my pov, it definitely requires a review for language. But technical wise, seems good to me. What do you think @KianYang-Lee ?

**Review by KianYang-Lee — APPROVED** [review]
Seems very good to me thanks for good job @evanyeohboonkhai , there are some repetitions for example sampling that was covered in Day 1 Pandas for Time Series notebook though, not sure whether that's an issue? But definitely approving this. 💪

**Pull request merged**